app/src/main/res/values/strings.xml

```diff
@@ -1050,14 +1050,14 @@
     <string name="gz_bz2_compressed">%1$s compressed</string>
     <string name="latest_backup">Latest backup</string>
     <string name="pref_import_export_keystore">Import/export KeyStore</string>
-    <string name="pref_import_export_keystore_msg">Import/export Bouncy Castle KeyStore (BKS) internally used by App Manager.</string>
+    <string name="pref_import_export_keystore_msg">Import/export Bouncy Castle KeyStore (BKS) used internally by App Manager.</string>
     <string name="import_keystore">Import KeyStore</string>
-    <string name="confirm_import_keystore">Are you sure want to replace the existing KeyStore? This action cannot be undone.</string>
+    <string name="confirm_import_keystore">Replace the existing KeyStore? This action cannot be undone.</string>
     <string name="failed">Failed</string>
     <string name="pref_rules_msg">Instant blocking, import/export/remove rules, etc.</string>
     <string name="saf">SAF</string>
-    <string name="pref_layout_orientation">Layout orientation</string>
-    <string name="orientation_follow_locale">Follow locale</string>
-    <string name="orientation_left_to_right">Left to right</string>
-    <string name="orientation_right_to_left">Right to left</string>
+    <string name="pref_layout_orientation">Text direction</string>
```

Comment:
Not 100% sure, and this is probably not technically correct. It is however more clear.
Layout orientation sounds like it changes to top-to-bottom, or the aspect ratio to me.